### PR TITLE
Alec/tfac 460 - Small changes to make mission hub ui work, add has seen squads functionality, and catch all little fixes for our overlap work

### PR DESCRIPTION
**Pull request by amaliwanag2**
this pr does the following: @jedtan @kmjennison 
- updates fixture data to reflect our new tabstreak and tab count datashapes
- adds a has seen squads field to user and adds logic to set it accordingly when a user is created
- fixes small issues with log tab mutation
- cleans up schema
-

### CODE REVIEW

**Comment by amaliwanag2** [comment]
hold off, pushing some more little fies

File: graphql/database/users/createUser.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ const createUser = async (
         image: backgroundCatImage.image,
         timestamp: moment.utc().toISOString(),
       },
+      hasSeenSquads: !!missionId,
```
Comment (kmjennison) [inline graphql/database/users/createUser.js:76]: Nice approach to this!

File: graphql/database/missions/utils.js [hunk 2]
```diff
@@ -10,6 +11,18 @@ const REJECTED = 'rejected'
 const COMPLETED = 'completed'
 const STARTED = 'started'
 const override = getPermissionsOverride(MISSIONS_OVERRIDE)
+
+const buildUserIdUsernameMap = async userMissionDocuments =>
+  (await UserModel.getBatch(
+    override,
+    userMissionDocuments.map(user => user.userId),
+    { ProjectionExpression: 'id, username' }
```
Comment (kmjennison) [inline graphql/database/missions/utils.js:19]: 💯 on the use of ProjectionExpression here.

**Review by kmjennison — APPROVED** [review]
LGTM 🦊 Great work!

**Pull request merged**